src/cljc/rems/common/util.cljc

```diff
@@ -554,3 +554,15 @@
   (is (= [] (replace-key [] :a :b)))
   (is (= {:b 1} (replace-key {:a 1} :a :b)))
   (is (= {:a 1} (replace-key {:a 1} :b :c))))
+
+(defn update-present
+  "Like clojure.core/update, but does nothing if the key `k` does not exist in `m`."
+  [m k f & args]
+  (if (find m k)
+    (apply update m k f args)
+    m))
+
+(deftest test-update-present
+  (is (= {:a 1} (update-present {:a 1} :b (constantly true))))
+  (is (= {:a 1 :b true} (update-present {:a 1 :b 2} :b (constantly true))))
+  (is (= {:a 1 :b true} (update-present {:a 1 :b nil} :b (constantly true)))))
```

Comment:
this medley function looks like it implements the same functionality? http://weavejester.github.io/medley/medley.core.html#var-update-existing

Reply:
Good find, I'll add a ticket about refactoring it away. I merely moved it to the common now since it was left in only CLJ util.

Reply:
https://github.com/CSCfi/rems/issues/2866